docs/resources/certificate_authority.md

```diff
@@ -0,0 +1,223 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "AAA"
+layout: "aci"
+page_title: "ACI: aci_certificate_authority"
+sidebar_current: "docs-aci-resource-aci_certificate_authority"
+description: |-
+  Manages ACI Certificate Authority
+---
+
+# aci_certificate_authority #
+
+Manages ACI Certificate Authority
+
+
+
+## API Information ##
+
+* Class: [pkiTP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/pkiTP/overview)
+
+* Supported in ACI versions: 1.0(1e) and later.
+
+* Distinguished Name Formats:
+  - `uni/tn-{name}/certstore/tp-{name}`
+  - `uni/userext/pkiext/tp-{name}`
+
+## GUI Information ##
+
+* Locations:
+  - `Admin -> AAA -> Security -> Certificate Authorities`
+  - `Cloud Network Controller -> Administrative -> Security -> Certificate Authorities`
+
+## Example Usage ##
+
+The configuration snippet below creates a Certificate Authority with only required attributes.
+
+```hcl
+
+resource "aci_certificate_authority" "example" {
+  certificate_chain = <<EOT
+-----BEGIN CERTIFICATE-----
+MIICODCCAaGgAwIBAgIJAIt8XMntue0VMA0GCSqGSIb3DQEBCwUAMDQxDjAMBgNV
+BAMMBUFkbWluMRUwEwYDVQQKDAxZb3VyIENvbXBhbnkxCzAJBgNVBAYTAlVTMCAX
+DTE4MDEwOTAwNTk0NFoYDzIxMTcxMjE2MDA1OTQ0WjA0MQ4wDAYDVQQDDAVBZG1p
+bjEVMBMGA1UECgwMWW91ciBDb21wYW55MQswCQYDVQQGEwJVUzCBnzANBgkqhkiG
+9w0BAQEFAAOBjQAwgYkCgYEAohG/7axtt7CbSaMP7r+2mhTKbNgh0Ww36C7Ta14i
+v+VmLyKkQHnXinKGhp6uy3Nug+15a+eIu7CrgpBVMQeCiWfsnwRocKcQJWIYDrWl
+XHxGQn31yYKR6mylE7Dcj3rMFybnyhezr5D8GcP85YRPmwG9H2hO/0Y1FUnWu9Iw
+AQkCAwEAAaNQME4wHQYDVR0OBBYEFD0jLXfpkrU/ChzRvfruRs/fy1VXMB8GA1Ud
+IwQYMBaAFD0jLXfpkrU/ChzRvfruRs/fy1VXMAwGA1UdEwQFMAMBAf8wDQYJKoZI
+hvcNAQELBQADgYEAOmvre+5tgZ0+F3DgsfxNQqLTrGiBgGCIymPkP/cBXXkNuJyl
+3ac7tArHQc7WEA4U2R2rZbEq8FC3UJJm4nUVtCPvEh3G9OhN2xwYev79yt6pIn/l
+KU0Td2OpVyo0eLqjoX5u2G90IBWzhyjFbo+CcKMrSVKj1YOdG0E3OuiJf00=
+-----END CERTIFICATE-----
+EOT
+  name              = "test_name"
+}
+
+// This example is only applicable to Cisco Cloud Network Controller
+resource "aci_certificate_authority" "example_tenant" {
+  parent_dn         = aci_tenant.example.id
+  certificate_chain = <<EOT
```

Comment:
did you consider doing the second example with a file input example? https://developer.hashicorp.com/terraform/language/functions/file

Reply:
We decided not to go through with this..